docs/source/preflight/cli-usage.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Running Preflight checks
+description: Learn how to run preflight checks from the CLI
+---
+
+
+In order to check that a cluster and/or host meets the application's requirements prior to attempting an install, you are able to run preflight checks in advance.  Whilst this is built in to some applications, it's possible to run these via the CLI.
+
+## Usage
+
+In order to run, `preflight` needs a spec to understand what information to collect, and what to do with that information.  That spec can be a file, or hosted at a URL, located in an oci registry, or provided as stdin.
```

Comment:
> That spec can be a file, or hosted at a URL, located in an oci registry, or provided as stdin.

They can technically come from a secret in a cluster also, if you use `secret/secretname` (I think it might actually be `secret/namespace/secretname`? TBC, test that to be sure)

Reply:
I can't see a reason to encourage that at this stage, though it'll become apparent on the 'new' CLI.  Given the reason for this doc is for preflights prior to install, the secrets won't be there (or will be outdated on an upgrade).